apps/system-e2e/src/support/email-account.ts

```diff
@@ -78,9 +92,8 @@ const makeEmailAccount = async () => {
   }
   console.log('created new email account %s', testAccount.user)
   console.log('for debugging, the password is %s', testAccount.pass)
-
-  const userEmail = {
-    email: testAccount.user,
+  const userEmail: EmailAccount = {
+    email: testAccount.user as string,
```

Comment:
```suggestion
    email: testAccount.user,
```
This typing should be handled in `testAccount` creation, if possible.